<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="databaseTreeWidget">
       <column>
        <property name="text">
         <string>Databases</string>
        </property>
       </column>
      </widget>
      <widget class="QTabWidget" name="mainTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tableTab">
        <attribute name="title">
         <string>Data View</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="dataTableWidget"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="addRowButton">
             <property name="text">
              <string>Add Row</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteRowButton">
             <property name="text">
              <string>Delete Row</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveChangesButton">
             <property name="text">
              <string>Save Changes</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="sqlTab">
        <attribute name="title">
         <string>SQL Editor</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="sqlEditor"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="executeButton">
             <property name="text">
              <string>Execute</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="loadScriptButton">
             <property name="text">
              <string>Load Script</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewDatabase"/>
    <addaction name="actionOpenDatabase"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionCreateTable"/>
    <addaction name="actionModifyTable"/>
    <addaction name="actionDeleteTable"/>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>Admin</string>
    </property>
    <addaction name="actionUserManagement"/>
    <addaction name="actionPermissions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuAdmin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewDatabase"/>
   <addaction name="actionCreateTable"/>
   <addaction name="actionUserManagement"/>
  </widget>
  <action name="actionNewDatabase">
   <property name="text">
    <string>New Database</string>
   </property>
  </action>
  <action name="actionOpenDatabase">
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCreateTable">
   <property name="text">
    <string>Create Table</string>
   </property>
  </action>
  <action name="actionModifyTable">
   <property name="text">
    <string>Modify Table</string>
   </property>
  </action>
  <action name="actionDeleteTable">
   <property name="text">
    <string>Delete Table</string>
   </property>
  </action>
  <action name="actionUserManagement">
   <property name="text">
    <string>User Management</string>
   </property>
  </action>
  <action name="actionPermissions">
   <property name="text">
    <string>Permissions</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
